rails/etsy_product_catalog/app/views/categories/index.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1>Etsy Categories</h1>
+
+  <% @categories.each do |category| %>
+  
+    <ul>
+      <li><%= link_to cat_name.sub("_", " ").splitmap(&capitalize) category_path(category.id) %></li>
```

Comment:
cat_name is undefined, the sub/split/map stuff looks a lot like your to_title method, and you're missing a comma.